modules/book.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/prefer-default-export */
+// Class book, to define a book
+// eslint-disable-next-line max-classes-per-file
```

Comment:
- [ OPTIONAL ] I highly recommend not disabling the linters because you should just solve the issues not disable it so please try to solve it 😊💪🏻💯